src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -53,8 +53,8 @@
     <!-- .NET Framework needs to use the legacy gRPC library -->
     <PackageReference Include="Grpc" Version="2.46.6" />
     <PackageReference Include="Grpc.Core" Version="2.46.6" />
-    <PackageReference Include="Castle.Core" Version="3.3.0" />
-    <PackageReference Include="Castle.Windsor" Version="3.3.0" />
+    <PackageReference Include="Castle.Core" Version="4.4.1" />
```

Comment:
Anything to be concerned about with the major version revs?

Reply:
Yes, definitely! I'm not certain this current line of attack is going to work out, so I may end up reverting this change. (But I also think it would be good to keep our NuGet dependencies a little more current than 2019 versions!)